### PR TITLE
Patch status update

### PR DESCRIPTION
**Pull request by stevedicko**
Added new standard text at the top of all Service Scopes stating about required patch health.
Added section to monitoring FAQ with regards to the use of an agent.

### CODE REVIEW

File: articles/managed-services/man-monitoring-faq.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ UKCloud uses the same technologies we use in house to monitor events across our
 
 This service also leverages the benefits of AIOps (see below).
 
+## Do I need to instal an agent within my VM to enable this service?
```
Comment (SteveHallUKCLoud) [inline articles/managed-services/man-monitoring-faq.md:34]: typo - Install

**Review by SteveHallUKCLoud — CHANGES_REQUESTED** [review]

**Review by suehighmoor — APPROVED** [review]
LGTM

File: articles/managed-services/man-monitoring-faq.md (hunk 1 above)
Comment (suehighmoor) [reply articles/managed-services/man-monitoring-faq.md]: Fixed!

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

**Pull request merged**